src/main/kotlin/org/jitsi/jibri/webhooks/v1/WebhookClient.kt

```diff
@@ -83,6 +83,7 @@ class WebhookClient(
     }
 
     private val client = client.config {
+        expectSuccess = false
```

Comment:
Is this related to the Kotlin change?

Reply:
It was something I encountered from the ktor upgraded, I explained it in that commit message (easy to miss in github):
```
note that we now need to explicitly set "expectSuccess" to false to
disable the default response validator in the ktor client.  this was
always the desired behavior, but due to a bug
(https://youtrack.jetbrains.com/issue/KTOR-1412) the response validation
was not kicking in before.
```